### PR TITLE
Detailed page

### PR DESCRIPTION
**Pull request by regnisalram**

### CODE REVIEW

**Comment by regnisalram** [comment]
changed it up so detailedpage is now an activity, clicking on item starts it with the right object. pls pull?

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -24,6 +24,8 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".activity.DetailedPageActivity">
+        </activity>
     </application>
 
 </manifest>
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @regnisalram  new line

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/DetailedPageActivity.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    TextView article;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.detailed_page);
+
+        Intent i = getIntent();
+        NewsItem newsItem = (NewsItem) i.getSerializableExtra("newsItem");
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/DetailedPageActivity.java]: @regnisalram dont need intent for anything else, just chain it getIntent().getS...

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (regnisalram) [reply app/src/main/AndroidManifest.xml]: done

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/DetailedPageActivity.java (hunk 2 above)
Comment (regnisalram) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/DetailedPageActivity.java]: done

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java [hunk 3]
```diff
@@ -50,11 +51,13 @@ protected void onCreate(Bundle savedInstanceState) {
...
-
-       DaggerMockServerComponent.builder().build().inject(this);
     }
 
-    public void login(View view) {
+    public void login(View view){
+
+//        final ReaderApiInterface apiService = api.getClient().create(ReaderApiInterface.class);
+
+        MockServer apiService = new MockServer();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: @regnisalram use injected apiService isntead

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by adamgyulavari — APPROVED** [review]

**Comment by adamgyulavari** [comment]
@regnisalram resolve conflicts

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java (hunk 3 above)
Comment (regnisalram) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: done

**Comment by regnisalram** [comment]
everything should be ok now

**Pull request merged**